packages/core/scss/components/form/_input.scss

```diff
@@ -30,26 +29,27 @@
     -webkit-box-shadow: 0 0 0 1000px var(--theme-color-component-info) inset !important;
     -webkit-text-fill-color: var(--theme-input--color--autofill) !important;
     background-color: var(--theme-input--background--autofill) !important;
-    border: var(--theme-input--border-thickness, 1px) solid
-      var(--theme-input--border-color--autofill) !important;
+    border: var(--theme-input--border-thickness, 1px) solid var(--theme-input--border-color--autofill) !important;
     color: var(--theme-input--color--autofill) !important;
   }
 
   &::placeholder {
     color: var(--theme-input-hint--color);
   }
 
-  &.hover,
-  &:hover {
-    border-color: var(--theme-input--border-color--hover) !important;
-    background-color: var(--theme-input--background--hover);
+  &.hover, &:hover {
+    &:not(.read-only, .disabled, [readonly], [disabled]) {
+      border-color: var(--theme-input--border-color--hover) !important;
+      background-color: var(--theme-input--background--hover);
+    }
   }
 
-  &.focus,
-  &:focus {
-    outline: 1px solid var(--theme-color-focus-bdr);
-    outline-offset: var(--theme-input--focus--outline-offset);
-    border-color: var(--theme-input--border-color--focus) !important;
+  &.focus, &:focus {
+    &:not([readonly], .read-only, .disabled) {
```

Comment:
Should this not be the same line as [l41](https://github.com/siemens/ix/pull/1476/files#diff-61df4d0875e766a8739dd4e17cc1c27919572e1918cfa1676ad96a1cc169f2edR41)?